pkg/kudoctl/packages/install/package.go

```diff
@@ -44,10 +44,12 @@ func Package(
 
 	applyOverrides(&resources, instanceName, namespace, parameters)
 
-	if err := validateParameters(
-		*resources.Instance,
-		resources.OperatorVersion.Spec.Parameters); err != nil {
-		return err
+	if !options.SkipInstance {
```

Comment:
How about adding a comment on why this is needed. E.g. what is stated in the PR description?